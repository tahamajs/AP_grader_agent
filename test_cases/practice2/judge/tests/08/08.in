enter 1
reserve_table 1
reserve_table 2 2
reserve_table 3 2
reserve_table 4 2
show_table_info 2
reserve_table 5 2
reserve_table 6 2
reserve_table 8 2
reserve_table 7 2
show_table_info 2
reserve_table 9 2
reserve_table 10 2
show_table_info 2
exit 2
show_table_info 2
exit 5
show_table_info 2
exit 6
show_table_info 2
exit 3
show_table_info 2
enter 18
exit 8
enter 18
reserve_table 18 4
show_table_info 2
switch 7
show_table_info 2
exit 4
exit 9
exit 18
show_table_info 2
reserve_table 18 4
reserve_table 4 3
reserve_table 30 3
reserve_table 43 4
show_table_info 4
show_table_info 3
enter 29
exit 1
exit 4
reserve_table 22
exit 18
