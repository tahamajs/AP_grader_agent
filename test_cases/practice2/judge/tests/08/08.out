Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Paria sits at table 1
Amir Hossein sits at table 2
Parsa sits at table 2
Mehrad sits at table 2
Table ID: 2
People at the table: Amir Hossein, Mehrad, Parsa
Table remaining capacity: 1
Waiting queue length: 0
Fereshte sits at table 2
Amir Akbar enters the waiting queue of table 2
Taha enters the waiting queue of table 2
Mahdi enters the waiting queue of table 2
Table ID: 2
People at the table: Amir Hossein, Fereshte, Mehrad, Parsa
Table remaining capacity: 0
Waiting queue length: 3
Sadegh enters the waiting queue of table 2
Hamid enters the waiting queue of table 2
Table ID: 2
People at the table: Amir Hossein, Fereshte, Mehrad, Parsa
Table remaining capacity: 0
Waiting queue length: 5
Amir Hossein exits!
Table ID: 2
People at the table: Fereshte, Mahdi, Mehrad, Parsa
Table remaining capacity: 0
Waiting queue length: 4
Fereshte exits!
Table ID: 2
People at the table: Amir Akbar, Mahdi, Mehrad, Parsa
Table remaining capacity: 0
Waiting queue length: 3
Amir Akbar exits!
Table ID: 2
People at the table: Mahdi, Mehrad, Parsa, Taha
Table remaining capacity: 0
Waiting queue length: 2
Parsa exits!
Table ID: 2
People at the table: Mahdi, Mehrad, Sadegh, Taha
Table remaining capacity: 0
Waiting queue length: 1
Table 1: 3 0
Table 2: 0 1
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Taha exits!
Table 2: 0 0
Table 1: 3 0
Table 6: 3 0
Table 4: 2 0
Table 5: 3 0
Table 3: 2 0
Amin sits at table 4
Table ID: 2
People at the table: Hamid, Mahdi, Mehrad, Sadegh
Table remaining capacity: 0
Waiting queue length: 0
Mahdi switches seats with Amin!
Table ID: 2
People at the table: Amin, Hamid, Mehrad, Sadegh
Table remaining capacity: 0
Waiting queue length: 0
Mehrad exits!
Sadegh exits!
Amin exits!
Table ID: 2
People at the table: Hamid
Table remaining capacity: 3
Waiting queue length: 0
Amin sits at table 4
Mehrad sits at table 3
Mike sits at table 3
Mohsen enters the waiting queue of table 4
Table ID: 4
People at the table: Amin, Mahdi
Table remaining capacity: 0
Waiting queue length: 1
Table ID: 3
People at the table: Mehrad, Mike
Table remaining capacity: 0
Waiting queue length: 0
Table 1: 3 0
Table 2: 3 0
Table 5: 3 0
Table 3: 0 0
Table 6: 3 0
Table 4: 0 1
Paria exits!
Mehrad exits!
Kiana sits at table 2
Amin exits!
